packages/angular_devkit/build_angular/src/utils/process-bundle.ts

```diff
@@ -540,6 +541,13 @@ export async function inlineLocales(options: InlineOptions) {
       const setLocaleText = `var $localize=Object.assign(void 0===$localize?{}:$localize,{locale:"${locale}"});`;
       contentClone = content.clone();
       content.prepend(setLocaleText);
+
+      if (options.localeDataBasePath) {
+        const localeDataText = loadLocaleData(options.localeDataBasePath, locale, true);
+        if (localeDataText) {
+          content.prepend(localeDataText + ';');
+        }
```

Comment:
we should error if the localeData for a given locale was not found

Reply:
hmm. we do error above in i18n-options, so why this check?

Reply:
This was mainly meant as a proof of concept to ensure that the injection didn’t break anything else unintentionally.

I’m in the process of reworking this now that the FW side has solidified.  I should have a more complete version up later today.